internal/cli/users.go

```diff
@@ -112,3 +489,63 @@ func deleteUserBlocksCmd(cli *cli) *cobra.Command {
 
 	return cmd
 }
+
+func (c *cli) userPickerOptions() (pickerOptions, error) {
```

Comment:
It's not possible to use a picker with users, as there will usually be so many that it is impractical.